frontend/kubecloud/src/components/dashboard/EditClusterNodesDialog.vue

```diff
@@ -197,7 +199,24 @@ const addFormSshKey = ref<number|null>(null);
 const sshKeys = ref<any[]>([]);
 const sshKeysLoading = ref(false);
 const sshKeysError = ref('');
+const formValid = ref(false);
 
+const validateNodeName = (value: string) :string|boolean =>  {
+  const msg = required('Name is required')(value) || isAlphanumericExpectUnderscore('Node name can only contain letters, numbers, and underscores.')(value);
+  return msg ? msg : true;
+};
+const validateCPU = (value: string) :string|boolean =>  {
+  const msg = required('CPU is required')(value) || min('CPU must be at least 1',1)(+value);
+  return msg ? msg : true;
+};
+const validateRAM = (value: string) :string|boolean =>  {
+  const msg = required('RAM is required')(value) || min('RAM must be at least 0.5GB',0.5)(+value)|| max('RAM must be at most 32GB',32)(+value);
+  return msg ? msg : true;
+};
+const validateStorage = (value: string) :string|boolean =>  {
+  const msg = required('Storage is required')(value) || min('Storage must be at least 15GB',15)(+value)|| max('Storage must be at most 1000GB',1000)(+value);
+  return msg ? msg : true;
+};
 onMounted(async () => {
```

Comment:
Move all validation functions to validation.ts

Reply:
They are already there; those are just wrappers to meet the rules expected type. Later on, when we replace all text fields with input validators or v-text-field, we can have them in one place